symengine/fields.cpp

```diff
@@ -289,14 +289,14 @@ bool GaloisFieldDict::gf_is_sqf() const
     return monic.is_one();
 }
 
-std::vector<std::pair<GaloisFieldDict, integer_class>>
+std::vector<std::pair<GaloisFieldDict, unsigned>>
 GaloisFieldDict::gf_sqf_list() const
 {
-    std::vector<std::pair<GaloisFieldDict, integer_class>> vec_out;
+    std::vector<std::pair<GaloisFieldDict, unsigned>> vec_out;
     if (degree() < 1)
         return vec_out;
-    integer_class n = integer_class(1);
-    unsigned r = mp_get_si(modulo_);
+    unsigned n = 1;
+    unsigned r = mp_get_ui(modulo_);
```

Comment:
This is unrelated to the PR, but why is this? `modulo_` can be arbitrarily large right?